app/services/preserved_object_handler.rb

```diff
@@ -87,6 +88,7 @@ def check_existence
             else
               update_status(pres_copy, PreservedCopy::INVALID_MOAB_STATUS)
             end
+            moab_validated = true
```

Comment:
I'm thinking more "moab validation has occurred" -- not that it PASSED validation.  Therefore, it is correct that last_moab_validation was now even if it was a failure.

Reply:
Is it worth considering tracking the distinction between a validation occurring and validation passing in the variable name? So, instead of `moab_validated`, we call the variable something like `moab_validation_occurred`?

Reply:
note the distinction between `moab_validateD` and `moab_validateS`.   I would use the former to say "I performed the moab validation checks" and I would us the latter to say "this object passes validation."   Is that too subtle?

Reply:
The subtlety for me is that I would also naturally use the former to say "This object passed validation". I don't think it's too subtle with your distinction in mind, though.

Reply:
`ran_moab_validation`